pajbot/modules/default_chat_states.py

```diff
@@ -92,6 +94,10 @@ def on_stream_start(self, **rest) -> bool:
             log.warning("on_stream_start failed in DefaultChatStatesModule because bot is None")
             return True
 
+        channel_id = self.bot.streamer.id
+        bot_id = self.bot.bot_user.id
+        authorization = self.bot.bot_token_manager
+
```

Comment:
I think I'd prefer these variables be inlined (so use `self.bot.streamer.id` instead of `channel_id` in the `update_follower_mode()` call below)

Reply:
The main reason I made these into variables, is that we will need to use these in every single other function in the module. Otherwise we'll be repeating the exact same 3 variables in every function.

Reply:
I agree with Randers' take - as we don't make any modifications to the variable, I don't think this is any more or less clean, so just use `self.bot....` variables inline